gp-nested-forms/gpnf-override-parent-merge-tag-on-submission.php

```diff
@@ -122,9 +131,40 @@ function override_child_entry_input_value( $entry, $field, $child_form, $input_i
 			$child_entry = GFAPI::get_entry( $child_entry_id );
 			$value       = GFCommon::replace_variables( $default_value, $child_form, $child_entry );
 			GFAPI::update_entry_field( $child_entry_id, $input_id, $value );
+
+			// If because of the field update, any formula evaluation should be done on the entry.
+			$this->reprocess_form_calculations( $child_entry_id );
 		}
 
 	}
+
+	function reprocess_form_calculations( $entry_id ) {
+		$entry = GFAPI::get_entry( $entry_id );
+		$form  = GFAPI::get_form( $entry['form_id'] );
+
+		foreach ( $form['fields'] as $field ) {
+			// Only process calculation fields
+			if ( $field->enableCalculation && $field->calculationFormula ) {
+				$field_id = $field->id;
+
+				// Retrieve the formula for this field and process. First check for GPDTC calculations because of ':' calculations.
+				if ( is_callable( array( gp_date_time_calculator(), 'modify_calculation_formula' ) ) ) {
+					$formula = gp_date_time_calculator()->modify_calculation_formula( $field->calculationFormula, $field, $form, $entry );
+				}
+
+				// Process any other filter customization over it.
+				$formula = apply_filters( 'gform_calculation_formula', $formula, $form, $field, $entry );
+
+				// Process/Calculate the formula
+				$parsed_formula = GFCommon::replace_variables( $formula, $form, $entry, false, false, false, 'text' );
+				// phpcs:ignore Squiz.PHP.Eval.Discouraged
+				$calculated_value = eval( 'return ' . $parsed_formula . ';' );
```

Comment:
Directly callign the `eval` seems suspicious. We should run that through existing logic.

Here's how we recalc with another snippet: https://github.com/gravitywiz/snippet-library/blob/master/experimental/gpdtc-recalc.php